integration-tests/it-edge/src/main/resources/microservice.yaml

```diff
@@ -128,6 +136,8 @@ servicecomb:
           transport: rest
       paramCodecRestOnly:
         transport: rest
+      SpringMVCObjectParamTypeSchema:
```

Comment:
Why need add this configuration? JAX-RS do not add this but Spring MVC add this.

Reply:
quryObjectTest in  Spring MVC doesn't support highway